infoblox/hlinfoblox.py

```diff
@@ -88,14 +89,29 @@ def convert_lease_to_fixed_address(self, address, fqdn=None,
         for name in names:
             print("Host Name [%s]" % (name))
             host_record = self.api.get_host(name, notFoundFail=False)
-            a_record = self.api.get_a_record_by_ip(address)
             print("Host Record [%s]" % (host_record))
-            print("A record [%s]" % (a_record))
-            if host_record is None and a_record is None:
-                print("    Cannot find record:host or a:record for [%s], exit" % (name))
-                # self._create_host_record(address, fqdn, mac)
+
+            if host_record:
+                dhcp_cfg = str(host_record['ipv4addrs'][0]['configure_for_dhcp'])
+            
+                if dhcp_cfg == 'True':
+                    print(" Host record for [%s] already exist" % host_record)
+                    break
+
+                else: 
+                    print("    Updating host record - converting lease to fixedaddress")
+                    for host_host_ipv4addr in host_record['ipv4addrs']:
```

Comment:
You're not using `host_host_ipv4addr`. Also, much better to make this variable name something simpler and more to the point, like `ipv4addr`.